install/gcp/scripts/initialize_forseti_services.sh

```diff
@@ -28,7 +28,7 @@ if ! [[ -f $FORSETI_SERVER_CONF ]]; then
     exit 1
 fi
 
-SQL_SERVER_LOCAL_ADDRESS="mysql://forseti-db-admin@127.0.0.1:${SQL_PORT}"
+SQL_SERVER_LOCAL_ADDRESS="mysql://root@127.0.0.1:${SQL_PORT}"
```

Comment:
Can you put a comment here that there is an issue to create a non-root user, and link to the the issue?